.buildkite/hooks/pre-command

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env bash
-
+# this script is executed by multiple buildkite agents running on one node, so we write the pid to a file
```

Comment:
The multiple agents are running as containers, why would be facing that scenario?